ci/pod/docker-compose.plugin.yml

```diff
@@ -187,18 +187,6 @@ services:
     networks:
       opa_net:
 
-  # Splunk HEC Logging Service
-  splunk:
```

Comment:
why remove this component?

Reply:
Because we don't need a separate splunk server now. Vector does the job

Reply:
ok